pubspec.yaml

```diff
@@ -29,16 +29,20 @@ dependencies:
   flutter_localizations:
     sdk: flutter
   freezed_annotation: ^2.0.3
-  go_router: ^4.1.0
+  go_router: ^6.0.2
   intl: ^0.17.0
-  json_annotation: ^4.5.0
-  package_info_plus: ^1.4.2
-  permission_handler: ^8.3.0
+  json_annotation: ^4.7.0
+  package_info_plus: ^3.0.2
+  permission_handler: ^10.2.0
   retrofit: ^3.0.1+1
+  injectable: ^1.5.0
+  equatable: ^2.0.0
+  flutter_riverpod: ^2.1.3
```

Comment:
@nmint8m Do you have the plan to go with `river_pod` as state management? If yes, you have to take more time to learn what it is and how it works. It would be more complex since you're a newbie. I suggest removing this so far, you will have a chance to do the migration after that 🙏

Reply:
Yes, I do use riverpod. I included it inside PR #59.